src/pages/dongwheeKim/Login/Login.scss

```
@@ -1 +1,100 @@
-// a
+.LoginDongWhee {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(250, 250, 250);
+
+  .loginLayout {
+    width: 350px;
+    border: 1px solid rgb(219, 219, 219);
+    background-color: rgb(255, 255, 255);
+
+    .logoName {
+      display: flex;
+      justify-content: center;
+      margin: 22px auto 12px;
+      font-family: 'Lobster';
+      font-size: 30px;
+    }
+
+    .loginValueWrapper {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-around;
+      height: 200px;
+      margin-bottom: 50px;
+      margin-top: 40px;
+      align-items: center;
+
+      input {
+        display: flex;
+        align-items: center;
+        padding: 10px;
+        margin-bottom: 5px;
+      }
+
+      input::placeholder {
+        color: rgb(214, 214, 214);
+        font-size: 12px;
+      }
+
+      input:hover {
+        border-color: #b9b9b9 #b9b9b9 #b9b9b9 #b9b9b9;
+      }
```

부모 선택자(&)를 이용해서 다음과 같이 작성할 수 있습니다.

```suggestion
      input {
        display: flex;
        align-items: center;
        padding: 10px;
        margin-bottom: 5px;
        
        &::placeholder {
          color: rgb(214, 214, 214);
          font-size: 12px;
        }
        
        &:hover {
          border-color: #b9b9b9 #b9b9b9 #b9b9b9 #b9b9b9;
        }
      }
```

---

오 scss 기능에 이런 게 있었나보군요!! 꿀팁 감사합니다!!